ansible_base/jwt_consumer/hub/auth.py

```diff
@@ -1,15 +1,33 @@
+import logging
+
 from ansible_base.jwt_consumer.common.auth import JWTAuthentication
 from ansible_base.jwt_consumer.common.exceptions import InvalidService
+from ansible_base.resource_registry.models import Resource
+
+logger = logging.getLogger('ansible_base.jwt_consumer.hub.auth')
 
 
 class HubJWTAuth(JWTAuthentication):
-    def process_permissions(self, user, claims, token):
+    def process_permissions(self, user, token):
         # Map teams in the JWT to Automation Hub groups.
         try:
-            from galaxy_ng.app.models.auth import Group
+            from galaxy_ng.app.models import Team
         except ImportError:
             raise InvalidService("automation-hub")
 
-        for team in claims.get("teams", []):
-            hub_group, _ = Group.objects.get_or_create(name=team["name"])
-            hub_group.user_set.add(user)
+        for role_name in token.get('object_roles', {}).keys():
+            if role_name.startswith('Team'):
```

Comment:
Error log for non-Team requests?

Reply:
Hub does not use the RBAC system to its fullest potential so for Hub we just ignore anything that is not a Team role.